explore/src/main/scala/explore/proposal/ProposalDetailsTile.scala

```diff
@@ -317,6 +319,14 @@ object ProposalDetailsBody:
       )
     }
 
+    extension (limits: SiteCoordinatesLimits)
+      def format: String =
+        val raStart  = limits.raStart.toHourAngle.toDoubleHours
+        val raEnd    = limits.raEnd.toHourAngle.toDoubleHours
+        val decStart = limits.decStart.toAngle.toSignedDoubleDegrees
+        val decEnd   = limits.decEnd.toAngle.toSignedDoubleDegrees
+        f"$raStart%.1f ≤ RA ≤ $raEnd%.1fh    $decStart° ≤ Dec ≤ $decEnd°"
```

Comment:
Will this produce decimals with any number of digits in `Dec`?

Reply:
Can this value be specified in sexagesimal anywhere (eg the API)?  If so, we might want to truncate somewhere. If "40°20'" is specified, it will be shown as 40.333333.......

Reply:
Hmmm. Yeah. The could specify in `hms` format. I'll force a maximum of one decimal, I guess.

Reply:
I chatted with Andy. He said we'll never need decimals in the Dec limits, so I'll just not print any, ever.